process.py

```diff
@@ -1223,21 +1305,25 @@ def build(release,package):
         # we need bug numbers so that we can commit our changes
         if get_need_zstream_clone(release) :
             # lookup cloned bug number
-            bugnumber,issue=issue_lookup(Jira,release,version,packages,True)
+            bugnumber,issue=issue_lookup(Jira,release,version,packages)
             if bugnumber == "0" :
-                print(">>>>parent bug not cloned yet");
                 entry['state']='waiting bug clone'
                 continue
             entry['bugnumber']=bugnumber
         else :
             # first lookup the bug to see if it has already been created
-            bugnumber,issue=issue_lookup(Jira,release,version,packages,False)
+            bugnumber,issue=issue_lookup(Jira,release,version,packages)
             if bugnumber == "0":
                 # nope, create it now
                 bugnumber,issue=issue_create(Jira,release,version,nss_version,firefox_version,packages)
+
                 if bugnumber == "0":
                     entry['state']='need bug'
                     continue
+
+                # Request clone right away
+                issue_request_clone(Jira, release, version, packages)
```

Comment:
You mentioned that with some rewriting the cloning could be put in a more optimal position. Since you have this patch mostly done and working (other then some minor comments I've had so far, got ahead and push this one when it's ready, but it's OK to do some rewrites if it makes future maintenance easier.

Reply:
I see we are ignoring the return code, so the fact that we are inconsistent in our return type doesn't matter for the functionality. Probably should make the returns consistent in either case.